docs/explanation/security-overview.rst

```diff
@@ -18,29 +18,5 @@ trusted.
 Cryptography
 ------------
 
-``imagecraft`` is a wrapper around several tools and mainly ``ubuntu-image``. The tool
-itself does not use cryptographic technologies. Refer to the `ubuntu-image security
-overview`_ for details on cryptographic technologies used in ``ubuntu-image``.
-
-
-Miscellaneous
-~~~~~~~~~~~~~
-
-- Secrets (passwords and hashes) can be present in the configuration files provided to
-  ``imagecraft`` to build images. Specifically, in the ``ubuntu-bootstrap`` plugin
-  configuration:
-
-  * In the ``extra-ppas`` customisation section, authentication tokens
-    (``user:password``) can be defined to access private PPAs. These values are used to
-    write the ``apt`` configuration without any treatment.
-  * In the ``manual`` customisation section, user accounts can be defined with plain
-    text or hashed passwords. These values are directly passed to the ``chpassword``
-    utility without any treatment.
-
-- These configuration files for secrets should then be securely stored, and if secrets
-  are used, they should ideally be injected at runtime.
-
-
-.. LINKS
-
-.. _ubuntu-image security overview: https://canonical-subiquity.readthedocs-hosted.com/en/latest/explanation/ubuntu-image-security-overview.html
+``imagecraft`` is a wrapper around several tools. The tool itself does not use
+cryptographic technologies.
```

Comment:
Double-checking: was this cleared with the relevant stakeholders?

Reply:
Imagecraft is not using `ubuntu-image` anymore. The `ubuntu-bootstrap` is also gone. Even if imagecraft's code is not doing any cryptographic operation, the craft framework probably do. I did not fully evaluated this, so what do you think of completely removing this `Cryptographic` section until I can properly assert that next cycle?

Reply:
Agreed, let's remove the section for now. I'm in discussion with Sergio about a good template we could use for all libraries that then gets transcluded in apps as-needed.